app/controllers/ApplicationController.scala

```diff
@@ -320,6 +320,22 @@ class ApplicationController @Inject() (implicit val env: Environment[User, Sessi
     }
   }
 
+  /**
+    * Returns the maintenance page.
+    */
+  def maintenance = UserAwareAction.async { implicit request =>
+    request.identity match {
+      case Some(user) =>
+        val timestamp: Timestamp = new Timestamp(Instant.now.toEpochMilli)
+        val ipAddress: String = request.remoteAddress
+
+        WebpageActivityTable.save(WebpageActivity(0, user.userId.toString, ipAddress, "Visit_Maintenance", timestamp))
+        Future.successful(Ok(views.html.maintenance("Project Sidewalk - Maintenance", Some(user))))
+      case None =>
+        Future.successful(Redirect("/anonSignUp?url=/maintenance"))
```

Comment:
We don't want to do this redirect for the maintenance page. No matter what, we just want to log to the `webpage_activity` table and then load the maintenance page. And if there is no signed in user, we can just grab our default "anonymous user" user ID. I would just rewrite the whole function like this:
```
def maintenance = UserAwareAction.async { implicit request =>
  val user: String = request.identity.map(_.userId.toString).getOrElse(UserTable.find("anonymous").get.userId)
  val timestamp: Timestamp = new Timestamp(Instant.now.toEpochMilli)
  val ipAddress: String = request.remoteAddress
  WebpageActivityTable.save(WebpageActivity(0, user, ipAddress, "Visit_Maintenance", timestamp))
  Future.successful(Ok(views.html.maintenance("Project Sidewalk - Maintenance", request.identity)))
}
```